### ISSUE TITLE
Allow to rename documents

### ISSUE THREAD

**Issue by tyll** [gobby/gobby#37]
If would be nice if it is possible to rename documents.

**Comment by aburgm** [comment]
I agree. There is actually some code in PR #7 for this, but I haven't heard from the author for a while now unfortunately.

**Comment by pintman** [comment]
Would be great to see this happen.

**Comment by fd0** [comment]
I've tested the code in #7, after polishing the patch a bit it works on gobby 0.5.0 and libinfinity 0.6.0. Would you be willing to merge this code if I port it to the latest master of libinfinity and gobby, respectively?

**Comment by aburgm** [comment]
@fd0 Yes, definitely interested in it. I'd be happy to review and eventually merge!

**Comment by Wiadomy** [comment]
Any news about this feature?

**Comment by fd0** [comment]
I haven't had the time to do this, sorry about that.
